src/components/Register.js

```diff
@@ -152,7 +152,7 @@ export const Register = () => {
             >
               {isSubmitting ? (
                 <>
-                  <CircularProgress size={14} />
+                  <Loader size={14} />
```

Comment:
you can actually keep `CircularProgress` here instead of using `Loader` since the previous implementation was not using a `div` with a `className` of `loading`